hcx-apis/src/main/java/org/swasth/hcx/controllers/v1/ParticipantController.java

```diff
@@ -60,6 +57,7 @@ public ResponseEntity<Object> participantCreate(@RequestHeader HttpHeaders heade
     @RequestMapping(value = "/search", method = RequestMethod.POST)
     public ResponseEntity<Object> participantSearch(@RequestBody Map<String, Object> requestBody) throws Exception {
         String url =  registryUrl + "/api/v1/Organisation/search";
+        validateParticipant(requestBody);
```

Comment:
validation should be added to participantUpdate, for participant search it is not required.

Reply:
done